mathics/core/parser/parser.py

```diff
@@ -840,20 +860,6 @@ def b_FractionBox(self, box1, token, p):
         box2 = self.parse_box(q + 1)
         return Node("FractionBox", box1, box2)
 
-    def b_FormBox(self, box1, token, p):
```

Comment:
Why was this removed?

Reply:
Because with the current logic, to implement the behavior of `FormBox` is not possible using this mechanism, so in my proposal, this method is never used.

Reply:
Yes, this is what I mean about the myopic kind of thing. FormBox needs to worry about precedence and the `b_` routines are where this kind of thing is done. So if you are defeating that, then maybe the thinking around this is flawed.

Reply:
Yes, I was aware about the possible issue with the precedence. So I checked it against WMA, and it also seems to neglect the operator precedence of ``` \` ```, so it gives the maximum precedence to it. On the other hand, the tokens that follows ``` \` ``` are processed using all the precedence rules. 


What makes me think that this is the right approach (or at least a better approach than the current one) is that it does not break the existing tests, and it just makes a difference if a ``` \`   ``` token  is found inside a ```LeftRowBracket```/```RightRowBracket``` expression.